Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.cpp

```diff
@@ -197,25 +197,25 @@ void HybridPICModel::InitData (const ablastr::fields::MultiFabRegister& fields)
     amrex::IntVect Ey_stag = fields.get(FieldType::Efield_fp, Direction{1}, 0)->ixType().toIntVect();
     amrex::IntVect Ez_stag = fields.get(FieldType::Efield_fp, Direction{2}, 0)->ixType().toIntVect();
 
-    // Check that the grid types are appropriate
-    const bool appropriate_grids = (
-#if   defined(WARPX_DIM_1D_Z)
-        // AMReX convention: x = missing dimension, y = missing dimension, z = only dimension
-        Ex_stag == IntVect(1) && Ey_stag == IntVect(1) && Ez_stag == IntVect(0) &&
-        Bx_stag == IntVect(0) && By_stag == IntVect(0) && Bz_stag == IntVect(1) &&
-#elif   defined(WARPX_DIM_XZ) || defined(WARPX_DIM_RZ)
-        // AMReX convention: x = first dimension, y = missing dimension, z = second dimension
-        Ex_stag == IntVect(0,1) && Ey_stag == IntVect(1,1) && Ez_stag == IntVect(1,0) &&
-        Bx_stag == IntVect(1,0) && By_stag == IntVect(0,0) && Bz_stag == IntVect(0,1) &&
-#elif defined(WARPX_DIM_3D)
-        Ex_stag == IntVect(0,1,1) && Ey_stag == IntVect(1,0,1) && Ez_stag == IntVect(1,1,0) &&
-        Bx_stag == IntVect(1,0,0) && By_stag == IntVect(0,1,0) && Bz_stag == IntVect(0,0,1) &&
-#endif
-        Jx_stag == Ex_stag && Jy_stag == Ey_stag && Jz_stag == Ez_stag
-    );
-    WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
-        appropriate_grids,
-        "Ohm's law E-solve only works with staggered (Yee) grids.");
+//     // Check that the grid types are appropriate
+//     const bool appropriate_grids = (
+// #if   defined(WARPX_DIM_1D_Z)
+//         // AMReX convention: x = missing dimension, y = missing dimension, z = only dimension
+//         Ex_stag == IntVect(1) && Ey_stag == IntVect(1) && Ez_stag == IntVect(0) &&
+//         Bx_stag == IntVect(0) && By_stag == IntVect(0) && Bz_stag == IntVect(1) &&
+// #elif   defined(WARPX_DIM_XZ) || defined(WARPX_DIM_RZ)
+//         // AMReX convention: x = first dimension, y = missing dimension, z = second dimension
+//         Ex_stag == IntVect(0,1) && Ey_stag == IntVect(1,1) && Ez_stag == IntVect(1,0) &&
+//         Bx_stag == IntVect(1,0) && By_stag == IntVect(0,0) && Bz_stag == IntVect(0,1) &&
+// #elif defined(WARPX_DIM_3D)
+//         Ex_stag == IntVect(0,1,1) && Ey_stag == IntVect(1,0,1) && Ez_stag == IntVect(1,1,0) &&
+//         Bx_stag == IntVect(1,0,0) && By_stag == IntVect(0,1,0) && Bz_stag == IntVect(0,0,1) &&
+// #endif
+//         Jx_stag == Ex_stag && Jy_stag == Ey_stag && Jz_stag == Ez_stag
+//     );
+//     WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
+//         appropriate_grids,
+//         "Ohm's law E-solve only works with staggered (Yee) grids.");
```

Comment:
Seems like I could have saved myself a lot of trouble here before by just checking `if (WarpX::GetInstance().grid_type == GridType::Staggered)` hehe